go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/MOHC-LTD/shopify v1.32.0
 	golang.org/x/time v0.0.0-20210611083556-38a9dc6acbc6
 )
+
+replace github.com/MOHC-LTD/shopify => "C:/Users/Rob G/dev/backend/shopify"
```

Comment:
Remember to change... 😳

Reply:
Yep, just waiting `shopify` to get published first 👍🏻